patches/master_patch.patch

```diff
@@ -2137,3 +2161,40 @@ index 9e744b9940e2c79366fae21f5652ee0fce830637..143385a2831ca5221a68b99bba65dc29
        ]
        #TODO(jrummell) Mac: 'DYLIB_INSTALL_NAME_BASE': '@loader_path',
      } else if (is_posix && !is_mac) {
+diff --git a/third_party/widevine/cdm/widevine.gni b/third_party/widevine/cdm/widevine.gni
+index 78695ef5572564001214133c10ef12ea92b3ad9b..a9606fca4d4d459df0e5027d6616315ccd9a5dde 100644
+--- a/third_party/widevine/cdm/widevine.gni
++++ b/third_party/widevine/cdm/widevine.gni
+@@ -2,6 +2,8 @@
+ # Use of this source code is governed by a BSD-style license that can be
+ # found in the LICENSE file.
+ 
++import("//build/config/chrome_build.gni")
++
+ declare_args() {
+   # Allow widevinecdmadapter to be built in Chromium.
+   enable_widevine = false
+@@ -11,6 +13,7 @@ template("widevine_sign_file") {
+   # For official builds, generate a signature file for |file| which will
+   # be used by Widevine. If |signature_file| is not specified, the signature
+   # file will be in the same directory as |file|.
++
+   action(target_name) {
+     forward_variables_from(invoker,
+                            [
+@@ -42,5 +45,15 @@ template("widevine_sign_file") {
+       "--flags",
+       "$flags",
+     ]
++    if (!muon_chromium_build) {
++      args += [
```

Comment:
also confused by this one because there is no source here for `signature_generator.py`

Reply:
we can't disclose the script from widevine. For now, we have to do manually copy to the destination. Maybe we can use submit it to our private repo and pull it to the right place